### ISSUE TITLE
console - IT tests - OrgsIT.createAndGet: Status expected:<200> but was:<500>

### ISSUE THREAD

**Issue by fvanderbiest** [georchestra/georchestra#2920]
https://travis-ci.org/georchestra/georchestra/builds/645967226
```
[ERROR] Failures: 
[ERROR]   OrgsIT.createAndGet:61 Status expected:<200> but was:<500>
[ERROR]   OrgsIT.createAndGetWithDescription:88 No value at JSON path "$.description": com.jayway.jsonpath.PathNotFoundException: Expected to find an object with property ['description'] in path $ but found 'java.lang.String'. This is not a json object according to the JsonProvider: 'com.jayway.jsonpath.spi.json.JsonSmartJsonProvider'.
[ERROR]   OrgsIT.createAndGetWithEmptyDescription:79 No value at JSON path "$.description": com.jayway.jsonpath.PathNotFoundException: Expected to find an object with property ['description'] in path $ but found 'java.lang.String'. This is not a json object according to the JsonProvider: 'com.jayway.jsonpath.spi.json.JsonSmartJsonProvider'.
[ERROR]   OrgsIT.createAndGetWithEmptyLogo:130 No value at JSON path "$.logo": com.jayway.jsonpath.PathNotFoundException: Expected to find an object with property ['logo'] in path $ but found 'java.lang.String'. This is not a json object according to the JsonProvider: 'com.jayway.jsonpath.spi.json.JsonSmartJsonProvider'.
[ERROR]   OrgsIT.createAndGetWithEmptyUrl:108 No value at JSON path "$.url": com.jayway.jsonpath.PathNotFoundException: Expected to find an object with property ['url'] in path $ but found 'java.lang.String'. This is not a json object according to the JsonProvider: 'com.jayway.jsonpath.spi.json.JsonSmartJsonProvider'.
[ERROR]   OrgsIT.createAndGetWithLogo:118 No value at JSON path "$.logo": com.jayway.jsonpath.PathNotFoundException: Expected to find an object with property ['logo'] in path $ but found 'java.lang.String'. This is not a json object according to the JsonProvider: 'com.jayway.jsonpath.spi.json.JsonSmartJsonProvider'.
[ERROR]   OrgsIT.createAndGetWithUrl:98 No value at JSON path "$.url": com.jayway.jsonpath.PathNotFoundException: Expected to find an object with property ['url'] in path $ but found 'java.lang.String'. This is not a json object according to the JsonProvider: 'com.jayway.jsonpath.spi.json.JsonSmartJsonProvider'.
[ERROR]   OrgsIT.deleteDescription:144 No value at JSON path "$.description": com.jayway.jsonpath.PathNotFoundException: Expected to find an object with property ['description'] in path $ but found 'java.lang.String'. This is not a json object according to the JsonProvider: 'com.jayway.jsonpath.spi.json.JsonSmartJsonProvider'.
[ERROR]   OrgsIT.deleteLogo:170 No value at JSON path "$.logo": com.jayway.jsonpath.PathNotFoundException: Expected to find an object with property ['logo'] in path $ but found 'java.lang.String'. This is not a json object according to the JsonProvider: 'com.jayway.jsonpath.spi.json.JsonSmartJsonProvider'.
[ERROR]   OrgsIT.deleteUrl:157 No value at JSON path "$.url": com.jayway.jsonpath.PathNotFoundException: Expected to find an object with property ['url'] in path $ but found 'java.lang.String'. This is not a json object according to the JsonProvider: 'com.jayway.jsonpath.spi.json.JsonSmartJsonProvider'.
[INFO] 
[ERROR] Tests run: 34, Failures: 10, Errors: 0, Skipped: 0
```

**Comment by fvanderbiest** [comment]
Wondering if this is related with #2908

**Comment by fvanderbiest** [comment]
Or https://github.com/georchestra/georchestra/blob/089dd7c70b8bc038daace636024a110e2cb86765/console/pom.xml#L878 fetching `georchestra/ldap:latest` rather than the one built on the same banch

**Comment by fvanderbiest** [comment]
Seems to be fixed by #2922
